x-pack/plugins/fleet/common/constants/agent_policy.ts

```diff
@@ -37,3 +37,10 @@ export const LICENSE_FOR_SCHEDULE_UPGRADE = 'platinum';
 export const DEFAULT_MAX_AGENT_POLICIES_WITH_INACTIVITY_TIMEOUT = 750;
 
 export const AGENTLESS_POLICY_ID = 'agentless'; // the policy id defined here: https://github.com/elastic/project-controller/blob/main/internal/project/security/security_kibana_config.go#L86
+
+export const agentLoggingLevels = {
```

Comment:
there is an existing constant with these values, should we move that to `common`?
https://github.com/elastic/kibana/blob/ce0162030beaa20008b92075671b1e1773e25a22/x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_logs/constants.tsx#L52-L57

Reply:
I can move it there, but I defined a new one because I need the keys to be displayed in the dropdown.